.classpath

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files:
1. look generated, and
2. don't seem to be obviously related to getting the two tests running that the PR title talks about.

As much as possible, try to keep PRs self-contained. If you want to make a change to the build system (which is what these appear to be...they look to me like they were generated by [Eclipse Buildship](https://github.com/eclipse-buildship/buildship), but I'm not 100% sure), that's fine, but it should be separated into a different PR than changes to functionality.

Reply:
If these files are generated by the build system, then they shouldn't be committed at all. I'm not familiar enough with Eclipse Buildship to know whether you're expected to commit these or not, but you should try to figure out what the intended practice is. For example, you might search stack overflow or read their documentation to find out.